package.json

```diff
@@ -112,6 +112,7 @@
     "identity-obj-proxy": "^3.0.0",
     "imports-loader": "^0.8.0",
     "jest": "^26.0.1",
+    "json-sort-cli": "^1.15.19",
```

Comment:
Awesome - we can probably remove the dependencies on the grunt tasks then?

Reply:
yes, I thought I did